src/transaction/boot_cl.h

```diff
@@ -76,4 +77,6 @@ extern void boot_clear_host_connected (void);
 extern char *boot_get_server_session_key (void);
 extern void boot_set_server_session_key (const char *key);
 extern int boot_add_collations (MOP class_mop);
+extern SERVER_TYPE get_conn_server_type ();
+extern void set_conn_server_type (SERVER_TYPE server_type);
```

Comment:
Rename using the boot_ suffix specific to the boot module:
```suggestion
extern SERVER_TYPE boot_get_conn_server_type ();
extern void boot_set_conn_server_type (SERVER_TYPE server_type);
```

Reply:
renamed